### PR TITLE
adds pulp streamer service

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

File: manifests/service.pp [hunk 1]
```diff
@@ -34,4 +34,12 @@
     status     => 'service pulp_resource_manager status | grep "node resource_manager"',
   }
 
+  service { 'pulp_streamer':
+    ensure     => running,
+    require    => [Service[mongodb]],
```
Comment (cristifalcas) [inline manifests/service.pp]: I'm not on the pulp team, but I would like to request to not force the pulp server to be on the same machine as the mongodb.

Comment (jlsherrill) [inline manifests/service.pp]: @cfouant should be able to check $manage_db and only require that if true.

File: manifests/apache.pp [hunk 2]
```diff
@@ -80,6 +80,16 @@
     }
   }
 
+  if $pulp::manage_db {
```
Comment (ehelms) [inline manifests/apache.pp]: Why manage_db?

File: manifests/apache.pp [hunk 3]
```diff
@@ -80,6 +80,16 @@
...
+    file {'/etc/pulp/streamer.conf':
```
Comment (ehelms) [inline manifests/apache.pp]: I think this should go in config.pp

Comment (ehelms) [inline manifests/apache.pp]: Actually, guessing this path is just wrong and you meant it to go in `/etc/httpd/conf.d` based upon the streamer.conf template

Comment (daviddavis) [inline manifests/apache.pp]: @ehelms is correct. this should be `/etc/httpd/conf.d/pulp_streamer.conf`.

File: manifests/service.pp (hunk 1 above)
Comment (jlsherrill) [inline manifests/service.pp]: @cristifalcas after some discussion, we do agree that the mongodb service should not be required for this service, its already required for all the other services so we should probably just address them all together.

Comment (daviddavis) [inline manifests/service.pp]: Added a redmine issue: http://projects.theforeman.org/issues/13864

**Comment by daviddavis** [comment]
Tested and it worked. ACK from me.

File: manifests/service.pp (hunk 1 above)
Comment (ehelms) [inline manifests/service.pp]: Looks like we can add this to (https://github.com/Katello/puppet-pulp/blob/master/manifests/database.pp#L14-L17) and remove these requires. I agree lets do a follow up with that change.

**Comment by ehelms** [comment]
Thanks @cfouant !

**Pull request merged**